docker/start.d/crond/220_configure_crond.sh

```diff
@@ -6,7 +6,7 @@ enable_cron_configs() {
     for i in $FILES_SORTED; do
         local filename=$(basename $i)
         sectionText "Enable cron config $filename"
-        ln -vs $i /var/spool/cron/crontabs/$filename &>> $BUILD_LOG
+        ln -vfs $i /var/spool/cron/crontabs/$filename &>> $BUILD_LOG
```

Comment:
looks good 👍 , I think it should be fine to use a force here - even if it might overwrite existing files as those would break how this concept is meant.

Thank you for the fix! 🙏